assets/js/custom/TranslateComments.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-env jquery */
+/* global Translation */
+/* global ByLineElementContainer */
+
+// eslint-disable-next-line no-unused-vars
+class TranslateComments extends Translation {
```

Comment:
The TranslateComments class handles the translation function specific to the comment section. It contains listeners for each of the translate and remove translation buttons for each comment. Then, it also contains methods that set, open, and close the translated content within each comment.